lib/ContractInteract/GatewayComposer.js

```diff
@@ -0,0 +1,440 @@
+'use strict';
```

Comment:
Same as BrandedToken:
* JSDoc must be improved, cleaned up, and aligned.
* Usage of `Promise.resolve()` unclear in `...RawTx` methods.